addon-test-support/fill-in.js

```diff
@@ -13,8 +14,19 @@ import wait from 'ember-test-helpers/wait';
 */
 export function fillIn(selector, text) {
   let el = getElementWithAssert(selector);
+
+  if (!isFocusable(el)) {
+    throw new Error('Unable to fill element');
+  }
+
   run(() => focus(el));
-  run(() => el.value = text);
+  run(() => {
+    if (el.contentEditable === 'true') {
+      el.textContent = text;
```

Comment:
seems like `textContent` is more suitable than `innerHTML`?

Reply:
Not sure. What happens if you pass a string containing html to this? `<p><strong>foo</strong>bar</p>`?
Does this treat is as a string or as HTML? 

And the other way around? If you use `innerHTML = "some text"`? Since "some text" is valid html, seems like the second would be more flexible.

Reply:
> Does this treat is as a string or as HTML?
I don't think so.

I think you are right.  `innerHTML` seems a better choice.